indexer-app/src/cmr/indexer/data/elasticsearch.clj

```diff
@@ -208,7 +210,17 @@
       (do
         (info "Ignoring update gran indexes request because gran index set is unchanged.")
         (info "Existing gran index set:" (pr-str existing-gran-index-set))
-        (info "New gran index set:" (pr-str expected-gran-index-set))))))
+        (info "New gran index set:" (pr-str expected-gran-index-set))))
+
+    ;; this is during our transition
+    (if (nil? existing-gran-index-set)
+      (let [_ (info "Creating gran index set for first time, using the old index-sets value if available.")
+            old-index-set (cmr.indexer.data.index-set-elasticsearch/get-old-index-set context es-config/gran-elastic-name idx-set/index-set-id)]
+        ;; put that value into the new index sets
+        (if (not (nil? old-index-set))
+          (do
```

Comment:
Replace (if (not with while-not  because a false condition doesn't exist.  Then you can also remove the do.

linter should have found this.